ext/project_data.yaml

```diff
@@ -33,21 +33,3 @@ gem_rdoc_options:
   - --main
   - README.md
```

Comment:
Can we delete lines 2-34, since all of that should be in `puppet.gemspec` now? It may be that `project: 'puppet'` is still needed, so ok if we leave that one